### DISCUSSION TITLE
Is the cloud offering primarily for CI/CD, or is it also a feature paywall?

### DISCUSSION THREAD

**Discussion by JohnYoungers** [ariga/atlas#2670] [Q&A] [answered]
For the beta features that require a login (e.g. triggers, functions, procedures), would they eventually be opened up once stable?  Or will there always be features of atlas that will always require a cloud membership?

**Comment by rotemtam** [comment, marked as answer]
Hi @JohnYoungers 

The answer is outlined in https://atlasgo.io/features

But briefly, advanced database features require login, unrelated to their status as beta or not.

We may release some of them to oss in the future, but no timeline is set for that.

Logging in to Atlas is free for end users and we intend to keep it that way. 

If there is some technicality that blocks you from using Atlas this way, please ping me or @a8m on Discord and we will try to help.

**Reply by JohnYoungers** [reply]
Thank you!  Sorry, I'm not sure how I missed that page

We're looking to start scaffolding applications using Postgres opposed to SQL Server, but a as a pre-requisite we'd really need a declarative replacement for our SqlProjects/dacpacs.  Given most of our projects do heavily rely on views/functions/stored procedures, it looks like without a cloud plan and using the oss version, this would need to partially be done via migrations

**Reply by a8m** [reply]
Just adding to this: views, triggers, functions, and the rest of the features went out of beta a few months ago.

**Reply by a8m** [reply]
> Given most of our projects do heavily rely on views/functions/stored procedures, it looks like without a cloud plan and using the oss version, this would need to partially be done via migrations

Correct. As Rotem mentioned, using these features (and other mentioned in the website) requires you to login to Atlas. We offer generous free tier that should be enough for small teams and individuals (3 seats, 2 databases and 1 project), but if you exceed this quota, you have the option to upgrade to the business or the enterprise tier, depending on your company's size - see the pricing page for more details: https://atlasgo.cloud/pricing.

Having said that, you are always welcome to use the community version of Atlas, manage these objects manually, and consider open a free account in the future or upgrading to one of the paid tiers if this justifies the ROI.

**Comment by jrschumacher** [comment]
The [free tier is no longer available](https://atlasgo.io/blog/2024/09/26/atlas-pro-pricing-update) as of September 27th, 2024. In addition, the declarative planning is not listed under the OSS feature set any longer.
